tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -2936,6 +2941,103 @@ def unique(self):
 
     return _UniqueDataset(self)
 
+  def rejection_resample(self,
+                         class_func,
+                         target_dist,
+                         initial_dist=None,
+                         seed=None):
+    """A transformation that resamples a dataset to achieve a target distribution.
+
+    Lets consider the following example where a dataset with an initial data
+    distribution of `init_dist` needs to be resampled into a dataset with
```

Comment:
since the parameter is called `initial_dist`, it would be good to also use `initial_dist` in the example, so that users don't accidentally set `init_dist` instead of `initial_dist`.

Reply:
addressed in the commit